pkg/requestutil/request_info.go

```diff
@@ -27,25 +27,37 @@ import (
 
 // RequestInfo holds service information from http.Request
 type RequestInfo struct {
-	ServiceLabel string
-	Method       string
-	Component    string
-	IP           string
-	TimeStamp    string
-	URLParam     string
-	BodyParam    string
+	ServiceLabel   string
+	Method         string
+	Component      string
+	IP             string
+	URLParam       string
+	BodyParam      string
+	StartTimeStamp int64
 }
 
 // GetRequestInfo returns request info needed from http.Request
 func GetRequestInfo(r *http.Request) RequestInfo {
 	return RequestInfo{
-		ServiceLabel: apiutil.GetRouteName(r),
-		Method:       fmt.Sprintf("%s/%s:%s", r.Proto, r.Method, r.URL.Path),
-		Component:    apiutil.GetComponentNameOnHTTP(r),
-		IP:           apiutil.GetIPAddrFromHTTPRequest(r),
-		TimeStamp:    time.Now().Local().String(),
-		URLParam:     getURLParam(r),
-		BodyParam:    getBodyParam(r),
+		ServiceLabel:   apiutil.GetRouteName(r),
+		Method:         fmt.Sprintf("%s/%s:%s", r.Proto, r.Method, r.URL.Path),
+		Component:      apiutil.GetComponentNameOnHTTP(r),
+		IP:             apiutil.GetIPAddrFromHTTPRequest(r),
+		URLParam:       getURLParam(r),
+		BodyParam:      getBodyParam(r),
+		StartTimeStamp: time.Now().Unix(),
+	}
+}
+
+// ExecutionInfo holds request execution info
+type ExecutionInfo struct {
+	EndTimeStamp int64
```

Comment:
How about making it simple as we only have one field here?

Reply:
has replaced struct to int64